engine/dlib/src/dlib/log.cpp

```diff
@@ -322,14 +350,17 @@ void LogInitialize(const LogParams* params)
         fprintf(stderr, "ERROR:DLIB: Unable to create @log message socket\n");
         if (message_socket != 0)
             dmMessage::DeleteSocket(message_socket);
-        dmSocket::Delete(server_socket);
+        if (server_socket != dmSocket::INVALID_SOCKET_HANDLE)
+            dmSocket::Delete(server_socket);
         return;
```

Comment:
Do we really need this `return` here?
Even if we can't create socket, we still need the log thread and working hooks.

Reply:
ah, it is message socket, ok, my mistake